File: drizzle/rhino/read/drizzle_read.js
#! /usr/bin/java org.mozilla.javascript.tools.shell.Main
// ------------------------------------------------------------------
//
//	drizzle_read.js
//
//						Jul/15/2014
//
// ------------------------------------------------------------------
importPackage(java.lang);
importPackage(java.sql);
load ("/var/www/data_base/common/rhino_common/sql_manipulate.js");

print	("*** 開始 ***");

var url = "jdbc:drizzle://localhost:4427/city";
var user = "scott";
var passwd = "tiger";

var conn = DriverManager.getConnection( url, user, passwd );

disp_proc (conn);

conn.close();

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: drizzle/clojure/read/drizzle_read.clj
; -----------------------------------------------------------------
;
;	drizzle_read.clj
;
;					Jul/17/2014
;
; -----------------------------------------------------------------
(load-file "/var/www/data_base/common/clojure_common/sql_manipulate.clj")

; -----------------------------------------------------------------
(use 'clojure.java.jdbc)
(println "*** 開始 ***")

(let [db-host "localhost:4427" db-name "city"]
	(def db {:classname "org.drizzle.jdbc.DrizzleDriver"
		:subprotocol "drizzle"
		:subname (str "//" db-host "/" db-name)
		:user "scott"
		:password "tiger"})
	(with-connection db
		(with-query-results rs
		["select id,name,population,date_mod from cities"]
			(display_proc rs)
		)
	))
(println "*** 終了 ***")
; -----------------------------------------------------------------
